lib/active_merchant/billing/gateways/cyber_source.rb

```diff
@@ -463,7 +463,7 @@ def add_address(xml, payment_method, address, options, shipTo = false)
           xml.tag! 'firstName',             payment_method.first_name             if payment_method
           xml.tag! 'lastName',              payment_method.last_name              if payment_method
           xml.tag! 'street1',               address[:address1]
-          xml.tag! 'street2',               address[:address2]                unless address[:address2].blank?
+          xml.tag! 'street2',               address[:address2]
```

Comment:
Do you know why this change was introduced in the first place? I was able to track it down to https://github.com/activemerchant/active_merchant/pull/376, which refers https://github.com/activemerchant/active_merchant/pull/329

I'd like to be sure it won't break anything

Reply:
I can see more similar checks in this method, that's a bit suspicious

Reply:
I recorded again VCR cassettes in https://github.com/chargify/chargify/pull/16701/files with empty street2 and everything works ok.

With `unless` you don't have a chance to remove saved street2 value.